### DISCUSSION TITLE
[ Question ] "Infinity Scrolling"

### DISCUSSION THREAD

**Discussion by hackermondev** [pixijs/pixijs#7179] [Q&A]
I'm creating a game with pixijs and I want to create something kind of like infinity scrolling so when the player gets to the edge I want the camera to move with the player basically making it like the room is infinity and it doesn't end.

Is this possible with PixiJS? and how?

**Comment by xzilja** [comment]
This library could be of great help I think https://github.com/davidfig/pixi-viewport they have this concept of moving view when mouse is close to screen edge, you could try to repurpose that logic and check when your character is close to screens edge perhaps?
